tests/test_pyxis_gql.py

```diff
@@ -109,6 +109,50 @@ def test_pyxis_graphql_get_repository_by_registry_path():
     assert repository == result["get_repository_by_registry_path"]["data"]
 
 
+def test_pyxis_graphql_find_repositories_by_repository_name():
```

Comment:
I don't mind if you don't add the unit tests for new APIs in pyxis_gql.py, the test cases (here and the existing ones) actually doesn't make much sense, APIs from pyxis_gql.py are wrappers of pyxis calls, we mocked the APIs to return the fixtures and then compare the result with fixtures which should always pass, you can just bypass these tests and focus on other parts.